### PR TITLE
Update README to reflect clone and mirror instructions from SOLAS.

### PR DESCRIPTION
**Pull request by davidewatson**

### CODE REVIEW

**Comment by davidewatson** [comment]
Sigh. There is no DO_NOT_MERGE label for this repo, so I can't use it.

The intent of this PR is to make all of our template repos (e.g. `solas`, `template-chart`, etc) follow the same conventions. We should probably let this percolate..

File: README.md [hunk 1]
```diff
@@ -23,15 +23,15 @@ rm -rf solas-chart.git
...
-Create a subdirectory with the name of your chart (eg sample-chart) in which
-you put the contents of your chart.
+Create a subdirectory with the name of your chart (eg sample-chart) in which you put the contents of your chart.
 
-# Edit README.md
+# What to do next?
 
-Edit README.md, changing the chart name in the first line and deleting
-everything below.
+- You should configure CI/CD by following the instructions for [GitHub](https://github.com/samsung-cnct/solas-chart/blob/master/docs/github.md), [Jenkins](https://github.com/samsung-cnct/solas-chart/blob/master/docs/jenkins.md), and [Quay](https://github.com/samsung-cnct/solas-chart/blob/master/docs/quay.md).
```
Comment (notjames) [inline README.md]: since "solas" was renamed to template, the links for solas-chart no longer works.

Comment (notjames) [inline README.md]: Are the docs in the Jenkins link relevant for Helm? This document works for Quay, but does it work for Helm? I'm checking, but I don't think it does.

Comment (notjames) [reply README.md]: OK...so just encapsulating the sidebar we just had. We'll need documentation that explains that the quay repo should be a quay application repo.

**Comment by davidewatson** [comment]
Superceeded by solas-chart.

**Pull request closed without merging**